checkov/terraform/checks/resource/oci/AbsSecurityGroupUnrestrictedIngress.py

```diff
@@ -0,0 +1,33 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class AbsSecurityGroupUnrestrictedIngress(BaseResourceCheck):
+    def __init__(self, check_id, port):
+        name = f"Ensure no security groups rules allow ingress from 0.0.0.0:0 to port {port}"
+        supported_resources = ['oci_core_network_security_group_security_rule']
+        categories = [CheckCategories.NETWORKING]
+        super().__init__(name=name, id=check_id, categories=categories, supported_resources=supported_resources)
+        self.port = port
+
+    def scan_resource_conf(self, conf):
+        direction = conf.get('direction')
+        source = conf.get('source')
+        protocol = conf.get('protocol')
+        tcp_options = conf.get('tcp_options')
+        if direction and direction[0] != 'INGRESS':
+            return CheckResult.PASSED
```

Comment:
```suggestion
            return CheckResult.UNKNOWN
```

Reply:
should i remove this case in the test file as well?

Reply:
leave the test case in the TF file, but of course you need to adjust the test class, because it won't flag it anymore